src/pages/HomePage/index.js

```diff
@@ -1,13 +1,25 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useIntl } from 'react-intl';
 
 import Map from 'components/common/Map';
 import HomeInfo from './HomeInfo';
 
 const HomePage = () => {
+  const [position, setPosition] = useState({ x: 51, y: -1 });
+  const intl = useIntl();
```

Comment:
I think this is not being used

Reply:
Yes that's right, it was left there because I thought of a confirmation popup asking for permission at first. Removed! ✓